src/client/components/Admin.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+// import './Admin.scss';
+
+class Admin extends React.Component {
```

Comment:
all stateless components should be functional components. `const Admin = (props: AdminProps) => {}`

Reply:
props and state for each component should also have TS types.